.gitignore

```diff
@@ -1,5 +1,5 @@
+**/.*/
```

Comment:
In my opinion is the best way to avoid auto generated folders in the git repo from IDEs or tools like .idea (webstorm) .code (vs code) .sass-cache, etc.